media/whip_connection.go

```diff
@@ -68,11 +71,18 @@ func (w *WHIPConnection) Stats() (*MediaStats, error) {
 	return p.Stats()
 }
 
+func (w *WHIPConnection) Kicked() bool {
+	w.mu.Lock()
+	defer w.mu.Unlock()
+	return w.kicked
+}
+
 func (w *WHIPConnection) Close() {
 	w.mu.Lock()
 	// set closed = true so getWHIPConnection returns immediately
 	// if we don't already have a peer - avoids a deadlock
 	w.closed = true
```

Comment:
maybe we can just rely on the `closed` bool but i saw that was also set here
https://github.com/livepeer/go-livepeer/blob/bbcc7f8fd0c1a9cb964721cac83d536999dbe7ed/media/whip_connection.go#L35-L41

Reply:
IIRC that does mean we shut things down before even bothering to create a peerconnection, so in that sense it has been "kicked"